test/manual/amp-carousel.amp.html

```diff
@@ -89,5 +89,11 @@
     <amp-img src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w400-h300-no" width=400 height=300></amp-img>
     <amp-img src="https://lh3.googleusercontent.com/Z4gtm5Bkxyv21Z2PtbTf95Clb9AE4VTR6olbBKYrenM=w400-h300-no" width=400 height=300></amp-img>
   </amp-carousel>
+
+  <amp-carousel id="carousel-4" width=400 height=300 type=slides controls indicators>
```

Comment:
Yes, it was "dots". We didn't quite remove it. For the context, we just didn't implement it because, with a slight tilt toward mobile and touch, the dots were a little less functional - I remember trying to tap some examples and it was very erratic in most places. But they are still good information-wise - i.e. they make it clear how many slides are out there. And obviously, they are just fine with mouse.

So, do you remember how we were planning to configure "dots"?